workspace_integration_test.go

```diff
@@ -1211,6 +1211,125 @@ func TestWorkspacesDeleteByID(t *testing.T) {
 	})
 }
 
+func TestCanForceDeletePermission(t *testing.T) {
+	client := testClient(t)
+	ctx := context.Background()
+
+	orgTest, orgTestCleanup := createOrganization(t, client)
+	defer orgTestCleanup()
+
+	wTest, _ := createWorkspace(t, client, orgTest)
```

Comment:
Hm, Is there a reason we don't clean this workspace up?

Reply:
I believe I was copying from the [update tests in this file](https://github.com/hashicorp/go-tfe/blob/main/workspace_integration_test.go#L740), which also ignore the cleanup

Im not sure why though...I will try to update all the tests to do the cleanup

Reply:
Oh, I remember. Because the the workspace gets removed in the process of the test, and then the cleanup call throws an errors. Or, in some other tests (and also this one, I would guess) the org also gets cleaned up first, which deletes the workspace and causes a defered workspace cleanup to fail

E.g. if you try to defer the cleanup in the test I linked above you get

```
=== RUN   TestWorkspacesUpdate
=== CONT  TestWorkspacesUpdate
    helper_test.go:1602: Error destroying workspace! WARNING: Dangling resources
        may exist! The full error is shown below.
        
        Workspace: 0874350d-2197-1355-74c7-4ee437bbc010
        Error: resource not found
--- FAIL: TestWorkspacesUpdate (5.32s)
```

Reply:
That feels like it should be a solve-able problem. I will update the workspace cleanup function not to error on 404s, and then update the other tests in this file to cleanup the workspaces which they create